### ISSUE TITLE
Release Planner app

### ISSUE THREAD

**Issue by dylanwauters** [microsoft/powerapps-tools#760]
When trying to import following ReleasePlanner.zip I get this error message: " There are missing dependencies. Install the following solutions before installing this one: "msdynce_ProductManagement (9.0.4.0066)" "

**Comment by ljrain** [comment]
Do you have Sales installed or just base Power Apps? Release Planner requires a Dynamics 365 Apps.

**Comment by BusinessNone** [comment]
> Do you have Sales installed or just base Power Apps? Release Planner requires a Dynamics 365 Apps. It makes use of the Product in the Product Catalog.

Why? It isn't like Product is a restricted entity. And so wouldn't this really then be a Dynamics 365 App instead of a Power App? Doesn't tying this to Dynamics 365 kind of go against the design principles?

**Comment by hitendra72** [comment]
An environment with Base/Vanilla CDS (i.e. no D365) does not include Products entity/table.

**Comment by ljrain** [comment]
> Why? It isn't like Product is a restricted entity. And so wouldn't this really then be a Dynamics 365 App instead of a Power App? Doesn't tying this to Dynamics 365 kind of go against the design principles?

I agree and the connection to Product is just a single lookup that could be removed.

**Comment by dylanwauters** [comment]
Another app we can't use.. why force people always to go in that D365 world --'
